main.go

```diff
@@ -47,39 +47,44 @@ func main() {
 			Aliases: []string{"c"},
 			Usage:   "This strategy generates list of engines using filters or specific keys and propose recommendations.",
 			Action: func(c *cli.Context) error {
```

Comment:
I'm lost in the logic above. While I can approve it - it should be refactored ASAP coz it's completely unclear.